Packs/Syslog/Integrations/SyslogSender/SyslogSender.yml

```diff
@@ -11,22 +11,35 @@ configuration:
   required: false
   type: 0
   section: Connect
-- defaultvalue: '514'
```

Comment:
Why did you remove the default?

Reply:
because when using TLS 6514 is the default port, so I added the default port while preparing the manager in the code- would you rather keep it anyway?